### ISSUE TITLE
Current 0.92 bin contains msvcp100.dll, but fstar.exe depends on msvcp110.dll. 

### ISSUE THREAD

**Issue by A-Manning** [FStarLang/FStar#636]
F\* depends on Z3, which depends on msvcp110.dll. 
Obvious fix is to include it in Fstar/bin.

**Comment by A-Manning** [comment]
@nikswamy @strub

**Comment by A-Manning** [comment]
Installing the Visual C++ 2012 Redistributable is also a fix. Consider listing as a dependency.

**Comment by s-zanella** [comment]
Closing in favor of #866.
